gnomad/sample_qc/pipeline.py

```diff
@@ -179,7 +180,10 @@ def get_qc_mt(
     )
 
     if ld_r2 is not None:
-        qc_mt = qc_mt.persist()
+        if checkpoint_path:
+            qc_mt = qc_mt.checkpoint(checkpoint_path)
+        else:
+            qc_mt = qc_mt.persist()
```

Comment:
I think we should also add a logger to this so users know we have a persist here if there isn't a checkpoint path.